### DISCUSSION TITLE
Milvus Datamigration from milvus to hdf5 or milvus to milvus for milvus version 2.2.13

### DISCUSSION THREAD

**Discussion by Yashwanth1712094** [milvus-io/milvus#33160] [Q&A and General discussion]
MIlvus version-2.2.13
Deployment mode(standalone or cluster): cluster/standalone

MQ type(rocksmq, pulsar or kafka): default/pulsar
SDK version(e.g. pymilvus v2.0.0rc2): pymilvus 2.2.13

Hello,
I've been trying to migrate data from milvus to milvus and milvus to hdf5 in both standalone and cluster mode.The current documentation(pymilvusdm==2.0) is asking for mysql parameters for meta data.i'm facing the issue as in  latest milvus meta data is stored in etcd.
Please suggest a way to migrate data between milvus to milvus or milvus to hdf5. Also please provide the link of official documentation for milvus dm for milvus version 2.2.13

**Comment by xiaofan-luan** [comment]
this is the way we implement to migrate between milvus cluster
https://github.com/zilliztech/milvus-migration?tab=readme-ov-file

**Reply by Yashwanth1712094** [reply]
Thank you for answer.
Does the tool supports migrating data from milvus2.x to milvus 2.x as in documentation  Milvus 2.x to Milvus 2.x is mentioned as future work.
Even there is no documentation for migrating data between milvus2.x and milvus 2.x.
If it does not support how to migrate data for the backup purpose.
